### DISCUSSION TITLE
Google Colab Links Not Working

### DISCUSSION THREAD

**Discussion by scottthomaswx** [community/community#124710] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

On Saturday google colab files saved in github worked fine. Today both in my and other repositories, google colab notebooks have "There's been an error" come up when you open them. Anyone else having this issue and/or know how to fix it?

**Comment by ben-williams-ai** [comment]
Yes, finding this on all colab files in any repo everywhere
